src/docdb/registerDocDBCommands.ts

```diff
@@ -46,12 +46,16 @@ export function registerDocDBCommands(): void {
 
     });
     registerCommand('cosmosDB.deleteDocDBDatabase', async (context: IActionContext, node?: DocDBDatabaseTreeItem) => {
+        const suppressCreateContext: ITreeItemPickerContext = context;
```

Comment:
It seems a little odd to declare a new variable for the context here. What if you change the `context` parameter type to `ITreeItemPickerContext` (which extends `IActionContext`). Then you can just set `context.suppressCreatePick = true`

Reply:
Related to [this comment](https://github.com/microsoft/vscode-cosmosdb/pull/1449#discussion_r422372415), Neelima's current method actually has better type validation. I would be totally fine with Will's suggestion if we turned on `strictFunctionTypes` in tsconfig. Wonder how hard that would be?

Reply:
Should we hold off on turning on strictFunctionTypes for this release? I am not sure what other parts of code will be affected by it

Reply:
I don't think we have time to mess with tsconfig before the next release